packages/stylelint-config-airbnb/index.js

```diff
@@ -0,0 +1,29 @@
+module.exports = {
+  plugins: [
+    'stylelint-scss',
+  ],
+  rules: {
+    'at-rule-empty-line-before': 'never',
```

Comment:
It's 3am and much wine so forgive me if I interpret this a little wrong 😏 

* The rule `at-rule-empty-line-before` your comment refers to is a core stylelint rule, it is **not** a rule from the `stylelint-scss` plugin, you can see a prefixed example of this rule for that plugin [here](https://github.com/airbnb/css/pull/23/files/e7956634495732478f2e76ada527e1ae80b6c997#diff-c8daf3b8e5463995761e914cf32b47a5R22)